MSBot/bin/msbot-connect-azure.js

```diff
@@ -98,7 +98,6 @@ async function processConnectAzureArgs(config) {
             endpoint: args.endpoint
         });
         config.connectService(endpointService);
-        process.stdout.write(`Connected ${endpointService.type}:${endpointService.endpoint}\n`);
         services.push(endpointService);
```

Comment:
Why are we removing this? I liked knowing when a service was successfully added to my .bot file..